db/db_impl.h

```diff
@@ -295,6 +295,9 @@ class DBImpl : public DB {
       const std::vector<std::string>& external_files,
       const IngestExternalFileOptions& ingestion_options) override;
 
+  virtual Status VerifyChecksum() override;
+  virtual Status VerifyChecksum(const std::string& file_path) override;
```

Comment:
what does the failure look like if the file doesn't exist or is deleted by flush/compaction while this function runs? you can also give guidance for users to disable file deletions to prevent sporadic failures.